sky/services/updater/update_service.mojom

```diff
@@ -0,0 +1,10 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module updater;
+
+interface UpdateService {
```

Comment:
Oops, pruned too many files. Added back.

Do I also need to check in the changes to sky/shell/BUILD.gn and sky/packages/sky_services/BUILD.gn that depend on this interface? Or is that part of step 2?